sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1578,7 +1578,9 @@ public partial class BlobLeaseClient
         protected virtual Azure.Storage.Blobs.BlobContainerClient BlobContainerClient { get { throw null; } }
         public virtual string LeaseId { get { throw null; } }
         public System.Uri Uri { get { throw null; } }
+        public virtual Azure.Response<Azure.Storage.Blobs.Models.BlobLease> Acquire(System.TimeSpan duration, Azure.RequestConditions conditions, Azure.RequestContext context) { throw null; }
```

Comment:
How do we determine that we want to add `RequestContext` to HLC methods vs using it internally.
I.e. why this instead of `AcquireIfNotYetAcuired` API that uses `RequestContext` internally?

Reply:
Because it's not really doing `AcquireIfNotYetAcquired`.  It's doing `AcquireIfBlobNotFoundOrContainerNotFoundOrLeaseAlreadyAcquired` and we don't want to add an overload for that specific, non-general scenario.